molecule/null-cr-values-test/kiali-cr.yaml

```diff
@@ -138,6 +138,7 @@ spec:
     metrics_port: null
     port: null
     web_fqdn: null
+    web_port: null
```

Comment:
I think you should move this (and also the other places) to appear after `web_history_mode` to keep alphabetization.
Right, @jmazzitelli ?

Reply:
correct... a peer settings within the same group should be sorted alphabetically.

Reply:
Can do, I kept them together because they are intrinsically related (changing both seems to be the correct default), but alphabetizing might be better.